tests/validator.test.js

```diff
@@ -659,75 +659,5 @@ tap.test('validate schema', async t => {
     t.end()
   })
 
-  await t.test('should not validate invalid new schema', t => {
```

Comment:
I would leave this test by chaning the expected result (validated) and with a property we expect to receive, such as `customSchema`